source/get-started/create-a-connection-string.txt

```diff
@@ -0,0 +1,66 @@
+.. _pymongo-get-started-connection-string:
+
+==========================
+Create a Connection String
+==========================
+
+You can connect to your MongoDB deployment by providing a
+**connection URI**, also called a *connection string*, which
+instructs the driver on how to connect to a MongoDB deployment
+and how to behave while connected.
+
+The connection string includes the hostname or IP address and 
+port of your deployment, the authentication mechanism, user credentials 
+when applicable, and connection options.
+
+.. TODO: Link to other connection options pages
+.. To connect to an instance or deployment not hosted on Atlas, see
+.. :ref:`Other Ways to Connect to MongoDB <pymongo-other-ways-to-connect>`.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Find your MongoDB Atlas Connection String
+
+      To retrieve your connection string for the deployment that
+      you created in the :ref:`previous step <pymongo-get-started-create-deployment>`,
+      log into your Atlas account and navigate to the
+      :guilabel:`Database` section and click the :guilabel:`Connect` button
+      for your new deployment.
+
+      .. figure:: /includes/figures/atlas_connection_select_cluster.png
+         :alt: The connect button in the clusters section of the Atlas UI
+
+      Proceed to the :guilabel:`Connect your application` section and select
+      "Python" from the :guilabel:`Driver` selection menu and the version
+      that best matches the version you installed from the :guilabel:`Version`
+      selection menu.
+
+      Select the :guilabel:`Password (SCRAM)` authentication mechanism.
+      
+      Deselect the :guilabel:`Include full driver code example` option to view
+      the connection string.
+
+   .. step:: Copy your Connection String
+
+      Click the button on the right of the connection string to copy it to
+      your clipboard as shown in the following screenshot:
+
+      .. figure:: /includes/figures/atlas_connection_copy_string_python.png
+         :alt: The connection string copy button in the Atlas UI
+
+   .. step:: Update the Placeholders
+
+      Paste this connection string into a file in your preferred text editor
+      and replace the "<username>" and "<password>" placeholders with
```

Comment:
There's a weird line break here in staging. not sure if this will fix it:

```suggestion
      and replace the
      "<username>" and "<password>" placeholders with
```

Reply:
Is the linebreak you're seeing a break after the first quotation mark in "<username>"? I'll try and experiment some more but looks like it didn't change